tests/sentry/tagstore/snuba/test_backend.py

```diff
@@ -1,71 +0,0 @@
-from __future__ import absolute_import
-
-
-import json
-import responses
-
-from sentry.utils import snuba
-from sentry.models import GroupHash
-from sentry.testutils import TestCase
-from sentry.tagstore.snuba.backend import SnubaTagStorage
-
-
-class TagStorage(TestCase):
```

Comment:
Dropped this mocked out test for the new `snuba` tests that actually hit CH.

Also `get_group_ids_for_search_filter` is not impl anymore.